chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
I thought we agreed to, yes. From @dstillman: "The drop-down should show the proper entries for the "de" and similar cases (by checking for "de-DE", etc.), and for locales we don't have names defined for ("sw-KE") it should probably just show them as-is at the top of the menu."